classes/controller/AdminController.php

```diff
@@ -2689,10 +2722,10 @@ public function setMedia($isNewTheme = false)
             if (defined('_PS_HOST_MODE_') && _PS_HOST_MODE_) {
                 $this->addJS('https://cdn.statuspage.io/se-v2.js');
 
-                Media::addJsDefL('status_operational', $this->l('Operational', null, true, false));
-                Media::addJsDefL('status_degraded_performance', $this->l('Degraded Performance', null, true, false));
-                Media::addJsDefL('status_partial_outage', $this->l('Partial Outage', null, true, false));
-                Media::addJsDefL('status_major_outage', $this->l('Major Outage', null, true, false));
+                Media::addJsDefL('status_operational', $this->trans('Operational', null, true, false));
```

Comment:
Because of js, add escape here:
```
protected function l($string, $class = null, $addslashes = false, $htmlentities = true)
```

Thanks @Quetzacoalt91

Reply:
I thought I had fixed those as well, my mistake